contracts/reserve-auction/src/execute.rs

```diff
@@ -79,113 +85,132 @@ pub fn execute_create_auction(
     env: Env,
     collection: Addr,
```

Comment:
Since we have to pass around `collection` and `token_id` around together in many places, maybe we should have a struct called `Nft` that includes both in the common code package. Could possibly remove the `too_many_arguments` with it.

Reply:
Hmm yea you are right, let me see...

Reply:
It looks like I would have to change the code in a lot of places, almost every message. Maybe it's ok as is?

Reply:
@tasiov Okay either way, but grouping related args can make code more readable and an `Nft` type is something that can be re-used in Marketplace and elsewhere.